File: backend/src/main/java/cn/yang/nebula/agent/business/file/space/dto/FileSpaceUpdateDto.java
package cn.yang.nebula.agent.business.file.space.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

import java.io.Serial;
import java.io.Serializable;

/**
 * 更新个人空间入参
 *
 * @author AntiGravity
 */
@Data
public class FileSpaceUpdateDto implements Serializable {

    @Serial
    private static final long serialVersionUID = 9119523780566702410L;

    /**
     * 个人空间 ID
     */
    @NotBlank(message = "个人空间ID不能为空")
    private String id;

    /**
     * 名称
     */
    private String name;

    /**
     * 备注
     */
    private String remark;
}

File: frontend/src/type/filespace/FileSpaceUpdateDto.ts
/**
 * 修改文件空间入参
 */
export interface FileSpaceUpdateDto {
    /**
     * 空间ID
     */
    id: string;

    /**
     * 空间名称
     */
    name?: string;

    /**
     * 备注
     */
    remark?: string;
}
